docs/pages/resources/characters.fr.mdx

```diff
@@ -0,0 +1,180 @@
+# Personnages
+Cette ressource fournit des informations sur les personnages de l'Univers Harry Potter.
+
+## Attributs
+{
+<table>
+  <thead>
+    <tr>
+      <th>Nom</th>
+      <th>Type</th>
+      <th>Description</th>
+      <th>Filtrable</th>
+    </tr>
+  </thead>
+  <tbody>
+    {[
```

Comment:
Could you please fix the tabs / spacing of the brackets in thise file? See the other pages for reference.